src/main/java/database/Database.java

```diff
@@ -37,6 +42,35 @@ private Database() {
             String jdbcUrl = "jdbc:postgresql://" + hostname + ":" + port + "/" + dbName + "?user=" + userName + "&password=" + password;
             this.connection = DriverManager.getConnection(jdbcUrl);
             System.out.println("Database connected");
+
+            //Creating tables
+            Statement users = this.connection.createStatement();
+            String sqlUser = "CREATE TABLE USERS " +
+                         "(ID INT PRIMARY KEY   NOT NULL," +
+                         " NAME         VARCHAR(50)  NOT NULL, " +
+                         " USERNAME     VARCHAR(50)  NOT NULL, " +
+                         " PASSWORD     VARCHAR(50)  NOT NULL)";
+            users.executeUpdate(sqlUser);
+            users.close();
+
+            Statement commands = this.connection.createStatement();
+            String sqlCommands = "CREATE TABLE COMMANDS" +
+                                 "(ID INT PRIMARY KEY   NOT NULL," +
+                                 " GAMEID       INT     NOT NULL," +
+                                 " VERSION      INT     NOT NULL," +
+                                 " COMMAND      BLOB    NOT NULL)";
+            commands.execute(sqlCommands);
+            commands.close();
+
+            Statement games = this.connection.createStatement();
+            String sqlGames =   "CREATE TABLE GAMES" +
+                                "(ID INT PRIMARY KEY    NOT NULL," +
+                                " TITLE     VARCHAR(50) NOT NULL," +
+                                " STATE         BLOB    NOT NULL)";
+            games.execute(sqlGames);
+            games.close();
+            System.out.println("Tables initialized");
+
         } catch (ClassNotFoundException | SQLException e) {
```

Comment:
Table names and column names should all be in PascalCase.  That would need to be changed in the other queries as well.